blueocean-core-js/src/js/fetch.js

```diff
@@ -5,6 +5,9 @@ import utils from './utils';
 import config from './config';
 import dedupe from './utils/dedupe-calls';
 import urlconfig from './urlconfig';
+import { prefetchdata } from './scopes';
+
+const Promise = es6Promise.Promise;
```

Comment:
ah right. I don't think you need to do that? At the very least i have used the contructor in other places without it. But I don't thing it hurts also so meh.